addons/website_forum/static/src/js/follow_tags.js

```diff
@@ -0,0 +1,64 @@
+odoo.define('website_forum.follow_tags', function (require) {
+'use strict';
+
+var ajax = require('web.ajax');
+var animation = require('web_editor.snippets.animation');
+
+animation.registry.follow_tags = animation.Class.extend({
+    selector: ".o_tags_follow",
+    start: function(editable_mode) {
+        var self = this;
+        if (editable_mode) {
+            self.$target.find(".o_js_forum_tag_follow").off('mouseenter mouseleave');
+        }
+        self.is_user = self.$target.data('is_user') || false;
+        self.email = self.$target.data('email') || '';
+        self.$target.find(".o_js_forum_tag_follow").each(function() {
+            var tag_id = $(this).find('a').data('tag-id');
+            var follow = self.$target.find("div[data-ids=" + tag_id + "]").data('follow');
+            var is_follow = (follow == 'on')?true:false;
+            self.toggle_subscription(tag_id, is_follow, self.email);
```

Comment:
simplify: use follow === 'on' instead of "is_follow" and remove the "var is_follow..."